FeedStoreChallenge/CoreDataFeedStore/Model/CDFeedImage+Helpers.swift

```diff
@@ -0,0 +1,28 @@
+// 
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+extension CDFeedImage {
+	func toLocalFeedImage() -> LocalFeedImage? {
```

Comment:
The return can now be non-optional

```suggestion
	func toLocalFeedImage() -> LocalFeedImage {
```